src/pull-request/pull-request-general-information.html

```diff
@@ -24,39 +25,41 @@
       }
       .pr-topinfo {
         display:flex;
+        padding: 24px;
       }
       .pr-right-side {
         display:block;
       }
-      .markdown-html img {
-        max-width: 100%;
-      }
       reviewers-list {
         max-width: 300px;
       }
+      pull-request-comments {
+        display: block;
+      }
+      marked-github-element a {
```

Comment:
It would be better to move these changes to `marked-github-element` as the code is in the light-dom there. This also makes it global to all markdown we use.